### PR TITLE
demo notebook added for model card - model registry integration

### PR DESCRIPTION
**Pull request by shrestha-bikash**
*Issue #, if available:*
https://issues.amazon.com/issues/f5959552-07f0-4864-955e-83f8a4ee28a7

*Description of changes:*
Added new notebook example for the new feature, Model card - Model registry integration. This notebook shows guides on how to create a model card by associating model registry's model packages into it.

*Testing done:*
NA

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Zhenshan-Jin — APPROVED** [review]

File: sagemaker_model_governance/model_card_with_model_package.ipynb [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+   "cell_type": "markdown",
+   "id": "ba5c2fdd",
+   "metadata": {},
+   "source": [
+    "## Notebook CI Test Results\n",
+    "\n",
+    "This notebook was tested in multiple regions. The test results are as follows, except for us-west-2 which is shown at the top of the notebook.\n",
+    "\n",
+    "\n",
+    "![This us-east-1 badge failed to load. Check your device's internet connectivity, otherwise the service is currently unavailable](https://h75twx4l60.execute-api.us-west-2.amazonaws.com/sagemaker-nb/us-east-1/NOTEBOOK_PATH)\n",
```
Comment (Zhenshan-Jin2) [inline sagemaker_model_governance/model_card_with_model_package.ipynb:365]: update `NOTEBOOK_PATH` to `sagemaker_model_governance|model_card_with_model_package.ipynb`

File: sagemaker_model_governance/model_card_with_model_package.ipynb [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f6a7daa0-105e-4976-9eb9-363e8c6f78c9",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "This notebook's CI test result for us-west-2 is as follows. CI test results in other regions can be found at the end of the notebook. \n",
+    "\n",
+    "![This us-west-2 badge failed to load. Check your device's internet connectivity, otherwise the service is currently unavailable](https://h75twx4l60.execute-api.us-west-2.amazonaws.com/sagemaker-nb/us-west-2/sagemaker_model_governance|model_card.ipynb)\n",
```
Comment (Zhenshan-Jin2) [inline sagemaker_model_governance/model_card_with_model_package.ipynb:24]: update `model_card.ipynb` to `model_card_with_model_package.ipynb`

File: sagemaker_model_governance/model_card_with_model_package.ipynb [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+    "    TrainingJobName=training_job_name\n",
+    ")\n",
+    "\n",
+    "model_data_url = training_job[\"ModelArtifacts\"][\"S3ModelArtifacts\"]\n",
+    "image = training_job[\"AlgorithmSpecification\"][\"TrainingImage\"]\n",
+    "\n",
+    "# model package request input object\n",
+    "create_model_package_input_dict = {\n",
+    "    \"ModelPackageGroupName\": model_package_group_name,\n",
+    "    \"ModelPackageDescription\": \"Test model package registered for integ test\",\n",
```
Comment (Zhenshan-Jin2) [inline sagemaker_model_governance/model_card_with_model_package.ipynb:245]: update the description to something relevant, like `Test model package to be registered in model card`

**Comment by atqy** [comment]
Completed in https://github.com/aws/amazon-sagemaker-examples/pull/4318 instead - closed

**Pull request closed without merging**